cea/resources/radiation_arcgis/radiation.py

```diff
@@ -5,6 +5,8 @@
 
 import datetime
 import os
+from datetime import timedelta
+from datetime import datetime as datetime2
```

Comment:
this seems ugly... why not just use `datetime.timedelta` and `datetime.datetime`? a little more typing, but tacking on a `2` at the end of a variable feels weird to me...

Reply:
done